src/components/property-tree/property-tree-utils.tsx

```diff
@@ -12,18 +12,18 @@ interface TreeAsset {
 
 const addChildrenAssets = (
   childAssets: Asset[] | ValidChildAsset[] | undefined,
+  assetGuid: string,
 ): TreeAsset[] => {
   const childrenAssets: TreeAsset[] = [];
 
   if (childAssets) {
     for (const [childAssetIndex, childAssetValue] of childAssets.entries()) {
-      if (childAssetIndex < 5) {
+      if (childAssetIndex < 3) {
```

Comment:
3 is a magic number. Could this be replaced with a variable? For example, MaximumVisibleAssets

Reply:
Will do

Reply:
Added variable, with a value of 3

Reply:
Did you push the change? I cant see it

Reply:
Yeah the changes were pushed yesterday
https://github.com/LBHackney-IT/mtfh-frontend-property/blob/7c0aa906e65ca213bb1de5bc8a4ca90ec06ccf04/src/components/property-tree/property-tree-utils.tsx